ios/Sources/TimetableFeature/List/TimetableLoaded.swift

```diff
@@ -6,11 +7,15 @@ import SwiftUI
 public struct TimetableLoadedState: Equatable {
     public var timetableItems: [AnyTimetableItem]
     public var selectedType: SelectedType
-    // TODO: Replace with detail state
-    public var detail: AnyTimetableItem?
+    public var selectedTimetable: AnyTimetableItem?
```

Comment:
Maybe this will be `TimetableDetailState?` type variable. I think.
How about this?

I think this todo comment should be remaining.